src/components/common/Modal/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useRef } from 'react'
+import { isBrowser } from '../../../utils/detector'
+import { ReactComponent as CloseSvg } from '../../../img/close.svg'
+import styles from './Modal.module.scss'
```

Comment:
Please add some reasonable padding to modal content so it has a little more "space"